examples/go-client/service/service-poc.go

```diff
@@ -0,0 +1,272 @@
+// +build sample_client
+
+package service
+
+import (
+	"encoding/json"
```

Comment:
Imports should probably be organized by standard packages, packages in this repo, and third-party packages. You can use blank lines to separate those groups of imports.